extensions/AntiSpoof/AntiSpoofHooks.php

```diff
@@ -123,4 +123,21 @@ public static function asAddRenameUserHook( $uid, $oldName, $newName ) {
 		$spoof->update( $oldName );
 		return true;
 	}
+
+	/**
+	 * Wikia Addition
+	 * (After a successful user rename using Wikia Tool)
+	 *
+	 * @param $dbw DatabaseBase
+	 * @param $uid
+	 * @param $oldusername string
+	 * @param $newusername string
+	 * @param $process
+	 * @param $tasks
+	 */
+	public static function asAfterWikiaRenameUserHook( $dbw, $uid, $oldusername, $newusername, $process, &$tasks ) {
+		$spoof = self::makeSpoofUser( $newusername );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)